recipes/dictdiffer/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "dictdiffer" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "a86cd3cd465220d6f8e7882215bd04080a7520dc363ec0dc8f6e39f9aa670ca4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - pytest-runner >=2.7
+  run:
+    - python
+
+test:
+  imports:
+    - dictdiffer
+  requires:
+    - coverage >=4.0
+    - isort >=4.2.2
+    - mock >=1.3.0
+    - pydocstyle >=1.0.0
+    - pytest >=2.8.0
+    - pytest-cache >=1.0
+    - pytest-cov >=1.8.0
+    - pytest-pep8 >=1.0.6
```

Comment:
Since the test is just importing the package, you can remove the test requirements above

Reply:
@isuruf Remove the `require` section at all?